airflow/www/templates/appbuilder/navbar.html

```diff
@@ -50,6 +50,13 @@
         {% include 'appbuilder/navbar_menu.html' %}
       </ul>
       <ul class="nav navbar-nav navbar-right">
+	<li class="active">
```

Comment:
Small nit - can you fix the indent please?

Reply:
thx, I didn't realize it until you said it, but it was indented as TAB. I fixed it to 2 indents!